incubator/eventrouter/templates/configmap.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 Heptio Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{template "fullname" .}}
+  namespace: {{default "kube-system" .Values.namespace | quote}}
+  labels:
+    release: {{ .Release.Name | quote }}
+    app: {{template "fullname" .}}
+    heritage: {{.Release.Service | quote }}
```

Comment:
`chart` label missing here. Please use `{{template "name" .}}` for the `app` label.

See https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/labels.md

Apply label changes everywhere.